src/wallet/walletutil.cpp

```diff
@@ -60,10 +61,16 @@ std::vector<fs::path> ListWalletDir()
     const size_t offset = wallet_dir.string().size() + 1;
     std::vector<fs::path> paths;
     boost::system::error_code ec;
+    boost::system::error_code eci;
 
-    for (auto it = fs::recursive_directory_iterator(wallet_dir, ec); it != fs::recursive_directory_iterator(); it.increment(ec)) {
+    for (auto it = fs::recursive_directory_iterator(wallet_dir, ec); it != fs::recursive_directory_iterator(); it.increment(eci)) {
```

Comment:
I don't see why we can't reuse `ec`? The initialisation and increment should never run in the same iteration.

Reply:
Oh, it's so the error message can be different... nm